### PR TITLE
Feature: add to wishlist/like-video

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
### feature
- like btn click to add/remove
- watch later click to add/remove
- create global context to manage state for watchlater/like-video
- add empty list comp

https://user-images.githubusercontent.com/60725346/161595162-aa0dfef1-2812-428c-8ba3-4abcfd15b22e.mov

### CODE REVIEW

File: src/components/card/card.css [hunk 1]
```diff
@@ -8,3 +8,12 @@
 .card-icon {
   font-size: 1.2rem;
 }
+.card-icon:hover {
+  background-color: #ffffff40;
```
Comment (Chiragmodi01) [inline src/components/card/card.css:12]: prefer using variables for colors.

File: src/components/card/card.js [hunk 2]
```diff
@@ -1,7 +1,52 @@
...
+import { findItem } from "../../utils/findItem";
+import { useState } from "react";
 import "./card.css";
 
 export const Card = ({ video }) => {
   const { title, anime, _id } = video;
+  const {
+    globalState: { likeVideo, watchLater },
+    globalDispatch,
+  } = useGlobal();
```
Comment (Chiragmodi01) [inline src/components/card/card.js]: - I'm not sure what 'UseGlobal' context is for, prefer using 'UseVideo' or something.

File: src/components/index.js [hunk 3]
```diff
@@ -3,3 +3,4 @@ export { Sidebar } from "./sidebar/sidebar";
 export { Card } from "./card/card";
 export { Chips } from "./filterChips/chips";
 export { chipsData } from "./filterChips/chipsData";
```
Comment (Chiragmodi01) [inline src/components/index.js:5]: seems like you haven't used 'chip' here, so can be removed

**Review by dusariudaykumar — APPROVED** [review]
- The UI Looks awesome
-  The code is clean and your file structure is also good.
-  Everything is well structured. Good Work!

File: src/pages/likedvideoPage/likedvideo.js [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+    globalState: { likeVideo },
+  } = useGlobal();
+
   return (
     <div>
       <Navbar />
       <section className="main-box">
         <Sidebar />
         <main className="main-product">
+          {likeVideo.length === 0 && <EmptyComp />}
```
Comment (Chiragmodi01) [inline src/pages/likedvideoPage/likedvideo.js]: You can name it 'EmptyLikesPage' or 'PagePlaceholder'  in general.

**Review by Chiragmodi01 — COMMENTED** [review]
- Clicking on like is not filling thumb color in 'likes' page.
- You can increase the margin gap in between side navbar and main listing page.

Overall looks good.

File: src/components/card/card.js (hunk 2 above)
Comment (harshitpaliwal95) [reply src/components/card/card.js]: fixed it

File: src/components/index.js (hunk 3 above)
Comment (harshitpaliwal95) [reply src/components/index.js:5]: its used in explore page

File: src/pages/likedvideoPage/likedvideo.js (hunk 4 above)
Comment (harshitpaliwal95) [reply src/pages/likedvideoPage/likedvideo.js]: thanks for the feedback i fixed it

**Pull request merged**